sources/packages/backend/apps/workers/src/controllers/application/application.controller.ts

```diff
@@ -392,4 +415,38 @@ export class ApplicationController {
       });
     }
   }
+
+  /**
+   * Adds study end date is past application exception if the funding was applied after the deadline.
+   * @param application application.
+   * @param exceptions exceptions.
+   */
+  private addStudyEndDateIsPastApplicationException(
+    application: Application,
+    exceptions: ApplicationDataException[],
+  ): void {
+    const studyEndDate = application.currentAssessment.offering.studyEndDate;
+    const isFundingAppliedAfterDeadline = isLessThanGivenWeeks(
```

Comment:
Not part of this PR effort, one day we should have this logic centralized and the modal displayed upon application submission based on an API request.